app/index.jsx

```diff
@@ -5,7 +5,12 @@ import MortgageDetailsContainer from './containers/MortgageDetailsContainer';
 
 const AppNavigator = createStackNavigator(
 	{
-		Home: MortgageFormContainer,
+		Home: {
+			screen: MortgageFormContainer,
+			navigationOptions: {
```

Comment:
Don't remove the navigaation. It's not good UX to have a navigation bar on one screen and not have it on the other. Different screens having different flows is a weird flow